models/routegroups_endpoint_url.go

```diff
@@ -0,0 +1,56 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"context"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+)
+
+// RoutegroupsEndpointURL routegroups endpoint URL
+//
+// swagger:model routegroups.EndpointURL
+type RoutegroupsEndpointURL struct {
```

Comment:
This model chose a different case `Routegroups` instead of `RouteGroups`

Reply:
Isn't that expected because the package is named `routegroups`?

Reply:
Hmm, the go client autogeneration is doing it then. I think it's fine - we don't have to worry too much about this till we decide to publish the go sdk officially. So, this is fine as-is.